test/test_nestedtensor.py

```diff
@@ -550,6 +550,43 @@ def test_dropout(self, device, dtype):
                     expect_tensor[j] /= 1.0 - p
         self.nt_equal(nt, expect)
 
+    @dtypes(torch.float)
+    def test_linear(self, device, dtype):
+        a = torch.randn(1, 2, device=device, dtype=dtype)
+        b = torch.randn(2, 2, device=device, dtype=dtype)
+        c = torch.randn(3, 2, device=device, dtype=dtype)
+        nt = torch.nested_tensor([a, b, c])
+
+        weight = torch.randn(2, 2, device=device, dtype=dtype)
+        bias = torch.randn(2, device=device, dtype=dtype)
+        # success case
+        torch.functional.F.linear(nt, weight, bias)
+
+        # invalid nested tensor dimension
+        msg = 'input.dim()'
```

Comment:
I was in a dog fight with this dumb message for like an hour. I kept trying to expand this string to the full string but it would say that the string != string even though they did, so idk if it was like a hidden character thing or something.

Reply:
Could be because parens have special meaning in regex?